src/zimscraperlib/image/conversion.py

```diff
@@ -2,7 +2,7 @@
 import pathlib
 from typing import Any
 
-import cairosvg.svg  # pyright: ignore[reportMissingTypeStubs]
```

Comment:
Is this OK across all the supported range?

Reply:
There is no more range in python-scraperlib given our latest decision, only 3.13 ATM.

Reply:
Oh sorry, you speak about cairosvg range, will check